scripts/linters/python_linter.py

```diff
@@ -320,6 +341,35 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from pylint error messages.
+
+        Args:
+            lint_messages: list(str). Messages returned by the python linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # Remove newlines and coverage report from the end of message.
+        # we need to remove last five items from the list becuase starting from
+        # end we have three lines with just newline characters, fourth line
+        # contains the coverage report i.e.
+        # Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)
```

Comment:
Assert that lines are as expected before removing them.

You might be able to do some of this stuff using a regex and capture group instead.

Reply:
@Hudda is this done?

Reply:
Yes, this is done, forgot to comment.